Makefile

```diff
@@ -17,4 +17,4 @@ build-local-image: build-binary
 # This builds an image and pushes it up to a registry. This can be used to push
 # to the test or production Docker Hub registry by specifying $DOCKER_HUB_REPO.
 build-and-push-image: build-binary
-	docker buildx build --platform linux/arm64/v8,linux/amd64 --tag $(DOCKER_HUB_REPO):$(VERSION) --tag $(DOCKER_HUB_REPO):latest --build-arg GIT_COMMIT_HASH=$(GIT_COMMIT_HASH) --push .
+	docker buildx build --platform linux/arm64/v8,linux/amd64 --tag $(DOCKER_HUB_REPO):$(GIT_COMMIT_HASH) --tag $(DOCKER_HUB_REPO):$(VERSION) --tag $(DOCKER_HUB_REPO):latest --build-arg GIT_COMMIT_HASH=$(GIT_COMMIT_HASH) --push .
```

Comment:
Is this intentional?

Reply:
Are we concerned about keeping tags clean for the public? The git hash is labeled already, but we need to go into the image in the docker hub UI to see it.

Reply:
Ah, I'll remove it. It's here because that's what I think what we should be doing for our internal releases (releasing to ECR), since pulling by git commit is better IMO and it's what the subgraph gateway is doing atm. Will have a PR for the internal stuff shortly.